src/GridList/GridTile.js

```diff
@@ -0,0 +1,142 @@
+// @flow weak
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from '../utils/customPropTypes';
+
+export const styleSheet = createStyleSheet('MuiGridTile', (theme) => {
+  return {
+    root: {
+      position: 'relative',
+      display: 'block',
+      height: '100%',
+      overflow: 'hidden',
+      fontFamily: theme.typography.fontFamily,
+    },
+    childImg: {
+      height: '100%',
+      transform: 'translateX(-50%)',
+      position: 'relative',
+      left: '50%',
+    },
+  };
+});
+
+/**
+ * ```sx
+ * <GridTile>
+ *   <img src="image.jpg" />
+ *   <GridTileTitlebar title="GridTile" />
+ * </GridTile>
+ * ```
+ */
+export default class GridTile extends Component {
+  static propTypes = {
+    /**
+     * Theoretically you can pass any node as children, but the main use case is to pass an img,
+     * in which case GridTile takes care of making the image "cover" available space
+     * (similar to `background-size: cover` or to `object-fit: cover`).
+     */
+    children: PropTypes.node.isRequired,
+    /**
+     * The CSS `className` of the root element.
+     */
+    className: PropTypes.string,
+    /**
+     * Width of the tile in number of grid cells.
+     */
+    cols: PropTypes.number,
+    /**
+     * The component used for the root node.
+     * Either a string to use a DOM element or a component.
+     */
+    component: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.func,
+    ]),
+    /**
+     * Height of the tile in number of grid cells.
+     */
+    rows: PropTypes.number,
+  };
+
+  static defaultProps = {
+    cols: 1,
+    rows: 1,
+    component: 'div',
+  };
+
+  static contextTypes = {
+    styleManager: customPropTypes.muiRequired,
+  };
+
```

Comment:
@rosskevin Thought I'd asked, but mustn't have clicked submit  - where am I supposed to put that?